src/content/turbosnap/turbosnap.mdx

```diff
@@ -15,9 +15,10 @@ TurboSnap is an advanced Chromatic feature that speeds up [UI Tests](/docs/test)
 
 ## How it works
 
-1. TurboSnap considers the Git changes between the current commit and the [ancestor build's](/docs/branching-and-baselines#find-the-ancestor-builds) commit.
-2. Using Webpack's dependency graph, TurboSnap identifies the individual story files affected by those changes.
-3. Chromatic utilizes this information to selectively snapshot the stories defined in those specific story files. It also snapshots any tests that were denied on the ancestor build.
+1. Chromatic determines the [ancestor builds](/docs/branching-and-baselines#find-the-ancestor-builds) associated with your commit.
+2. TurboSnap considers the git changes between the current commit and the ancestor build's commit.  If more than one ancestor build is found, TurboSnap considers the union of the git changes.
+3. Using the dependency graph generated by Webpack or Vite, TurboSnap identifies the individual story files affected by those git-identified changes.
```

Comment:
@tmeasday let me know if this works or if the language should be specified differently!

Reply:
Gonna go ahead and merge, but if this language needs to be adjusted, let me know, and I'll be happy to open another PR!